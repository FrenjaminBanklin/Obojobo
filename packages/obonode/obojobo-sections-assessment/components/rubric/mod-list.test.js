import React from 'react'
import renderer from 'react-test-renderer'
import ModList from './mod-list'

describe('ModList', () => {
	test('renders as expected', () => {
		const component = renderer.create(
			<ModList
				node={{
					data: {
						get: () => {
							return {}
						}
					}
				}}
			/>
		)
		const tree = component.toJSON()

		expect(tree).toMatchSnapshot()
	})

	test('ModList component', () => {
		const component = renderer.create(
			<ModList
				node={{
					data: {
						get: () => {
							return {}
						}
					}
				}}
			/>
		)
		const tree = component.toJSON()

		expect(tree).toMatchSnapshot()
	})
})
