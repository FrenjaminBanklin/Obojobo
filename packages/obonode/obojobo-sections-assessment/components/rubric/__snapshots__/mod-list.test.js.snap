// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModList ModList component 1`] = `
<div>
  <p
    contentEditable={false}
  >
    Mods:
  </p>
</div>
`;

exports[`ModList renders as expected 1`] = `
<div>
  <p
    contentEditable={false}
  >
    Mods:
  </p>
</div>
`;
